pytest/lib/cluster.py

```diff
@@ -836,28 +837,29 @@ def start_cluster(num_nodes,
     if not config:
         config = load_config()
 
-    if not os.path.exists(os.path.expanduser("~/.near/test0")):
+    dot_near = pathlib.Path.home() / '.near'
```

Comment:
What is this magic :)

Reply:
`pathlib.Path` overrides the `/` operator. For `path` being a `pathlib.Path`, `path / string` is morally equivalent to `pathlib.Path(os.path.join(path, string))`.